### PR TITLE
Add /plugins command

### PR DESCRIPTION
**Pull request by rjworks**
plz work hehe

### CODE REVIEW

File: gomint-server/src/main/java/io/gomint/server/plugin/SimplePluginManager.java [hunk 1]
```diff
@@ -647,6 +647,11 @@ public String getBaseDirectory() {
         return (T) this.installedPlugins.get(name);
     }
 
+    @Override
+    public Map< String, Plugin > getPlugins() {
+        return this.installedPlugins;
```
Comment (Soupertonic) [inline gomint-server/src/main/java/io/gomint/server/plugin/SimplePluginManager.java]: Please return an immutable map.

`ImmutableMap.copyOf(this.installedPlugins)`

**Review by Soupertonic — CHANGES_REQUESTED** [review]

File: gomint-server/src/main/java/io/gomint/server/plugin/SimplePluginManager.java (hunk 1 above)
Comment (rjworks) [reply gomint-server/src/main/java/io/gomint/server/plugin/SimplePluginManager.java]: oops my bad

**Review by Soupertonic — DISMISSED** [review]
LGTM!

**Comment by Soupertonic** [comment]
I ran a test build and the compilation did not succeed. You did not declare the import of `ImmutableMap`. Please add `import com.google.common.collect.ImmutableMap;` as first import statement.

**Comment by Soupertonic** [comment]
image
It seems to be working for me. The compilation error is resolved and it apparently works.

**Review by Soupertonic — APPROVED** [review]

**Pull request merged**